### ISSUE TITLE
Warnings when viewing plugins page on multisite site

### ISSUE THREAD

**Issue by ojohnny** [ojohnny/wp-fallback-plugins-dir#1]
If the site is part of a multisite network, then we currently might see this when viewing the Plugins:

```
Warning: fopen(/REDACTED/wp-content/plugins/fallback-plugin/wp-gatsby/wp-gatsby.php): failed to open stream: No such file or directory in /REDACTED/wp-includes/functions.php on line 6111
```

This does not seem to affect the functionality otherwise.  Possibly, this is from the `get_plugin_data($file)` call, but I cannot say why the `$file` path would be different when looking at the plugins than otherwise browsing the admin interface.

**Comment by ojohnny** [comment]
Oh, now I see that it is checking the `fallback-plugin/xxx`, which means that it did not detect that this was a "dummy placeholder" just to make sure we did not load this twice.  I will look into it when I have the time.

**Comment by ojohnny** [comment]
This is fixed in version 1.0.4.
